docs/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+# PR Details
+
+## Description
+
+<!--- Describe your changes in detail -->
+
+## Checklist:
+
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If a section applies, ensure you have met all of the associated checks -->
+
+- [ ] My changes require new environment variables:
+  - [ ] I have updated the docker-compose, K8s YAML, and all dependent deployment configuration files.
+- [ ] My changes require updates to the documentation:
+  - [ ] I have updated the documentation accordingly.
+- [ ] My changes require updates and/or additions to the unit tests:
+  - [ ] I have modified/added tests to cover my changes.
+  - [ ] All existing tests pass.

```

Comment:
Let's pull this one out to the root level, even if they don't believe they have anything that would require unit test updates let's have them verify the existing ones pass.